### PR TITLE
[WIP][FEATURE] Add Visual Studio Code setup guide

### PR DESCRIPTION
**Pull request by gilbertsoft**
I started on a guide to setup Visual Studio Code. To speedup the installtion and setup I'm also creating a Extension Pack including the most important Extensions for the TYPO3 development.

Both are currently WIP. Hopefully I can finish this the within the next days...

### CODE REVIEW

File: Documentation/Appendix/SpecificTopics/IdeVisualStudioCodeSetup.rst [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+* If you prefer using the Insider Builds use https://code.visualstudio.com/insiders
+
+Quick setup
+===========
+
+Press :kbd:`ctrl` + :kbd:`shift` + :kbd:`X` and type `typo3-pack` to find the
+`TYPO3 Extension Pack` which installs a selection of useful extensions for
+the contribution to TYPO3. Then click `Install` to install all of it.
+
+Alternatifly use `code --install-extension gilbertsoft.typo3-pack` respectifly
```
Comment (sypets) [inline Documentation/Appendix/SpecificTopics/IdeVisualStudioCodeSetup.rst]: typos: 
Alternatifly => Alternatively
respctivly => respectively

Maybe better: Alternatively, use the command prompt to install:  `code --install ..` for x or `code-insiders ...` for y.

File: Documentation/Appendix/SpecificTopics/IdeVisualStudioCodeSetup.rst [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+========================
+
+If you need to select something from the menu in Visual Studio Code, menu items
+are displayed like this: `File: Settings: ....`
+
+Installation
+============
+
+* The latest stable version you can find at https://code.visualstudio.com/Download
+* If you prefer using the Insider Builds use https://code.visualstudio.com/insiders
```
Comment (sypets) [inline Documentation/Appendix/SpecificTopics/IdeVisualStudioCodeSetup.rst]: - Does everyone know, what 'Insider Builds' is?
- "The latest stable version you can find". Better: "You can find the latest stable version at"

File: Documentation/Appendix/SpecificTopics/IdeVisualStudioCodeSetup.rst [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+PHP files
+---------
+
+`File: Settings: Editor: Code Style : PHP : Set From : Predefined Style`
+
+Choose PSR-1 / PSR-2
+
+
+.. image:: _assets/phpstorm-settings-codestyle-php-setfrom.png
```
Comment (sypets) [inline Documentation/Appendix/SpecificTopics/IdeVisualStudioCodeSetup.rst]: phpstorm image?

File: Documentation/Appendix/SpecificTopics/IdeVisualStudioCodeSetup.rst [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+and Php Inspections are not TYPO3 specific, but will show possible errors and 
+are strongly recommended when developing for the TYPO3 core.
+
+.. hint::
+   None of these Plugins are mandatory, check out what might be useful for yourself!
+
+Install Plugins: `File: Settings: Plugins`
+
+
+* `DynamicReturnTypePlugin
```
Comment (sypets) [inline Documentation/Appendix/SpecificTopics/IdeVisualStudioCodeSetup.rst]: The rest of this page is from the PhpStorm setup page and will be changed later?

**Comment by gilbertsoft** [comment]
@sypets thanks for your early review. currently i'm working with benji on the extensions for vs code. after this has finished i'll continue here and of course will drop the phpstorm related things, was just takken as draft.

**Comment by sypets** [comment]
@gilbertsoft Are you still working on this? If not, I would like to close it for now.

**Comment by sypets** [comment]
@gilbertsoft I am closing, just reopen or contact me on Slack, if you still want to work on this :smile:

**Pull request closed without merging**